src/translate_c.zig

```diff
@@ -4853,6 +4856,50 @@ fn transType(rp: RestorePoint, ty: *const clang.Type, source_loc: clang.SourceLo
     }
 }
 
+fn qualTypeWasDemotedToOpaque(c: *Context, qt: ZigClangQualType) bool {
```

Comment:
Wouldn't it be better to memoize this info?

Reply:
I doubt it'd make any noticeable difference. Most of the time this won't do more than a single switch and possibly a hashmap lookup which would happen anyways if memoized.